sessionctx/variable/tidb_vars.go

```diff
@@ -1044,6 +1051,10 @@ var (
 	PreparedPlanCacheMemoryGuardRatio = atomic.NewFloat64(DefTiDBPrepPlanCacheMemoryGuardRatio)
 	EnableConcurrentDDL               = atomic.NewBool(DefTiDBEnableConcurrentDDL)
 	EnableNoopVariables               = atomic.NewBool(DefTiDBEnableNoopVariables)
+	// EnableFastReorg indicates whether to use lightning to enhance DDL reorg performance.
+	EnableFastReorg = atomic.NewBool(false)
```

Comment:
```suggestion
	EnableFastReorg = atomic.NewBool(DefTiDBEnableFastReorg)
```

It is better to keep the style consistent with other variables.

Reply:
Done